runtime-transaction/benches/process_compute_budget_instructions.rs

```diff
@@ -17,6 +18,10 @@ use {
 const NUM_TRANSACTIONS_PER_ITER: usize = 1024;
 const DUMMY_PROGRAM_ID: &str = "dummmy1111111111111111111111111111111111111";
 
+fn feature_set() -> FeatureSet {
+    FeatureSet::all_enabled()
```

Comment:
same with these benches, shouldn't create the feature set hashmap on each bench iter